README.md

```diff
@@ -49,6 +49,33 @@ This list is just for Red Hat contributions to projects hosted on GitHub.  See h
 ### Hosting projects on /RedHatOfficial
 
 Open an issue in *this* repo, and the owners of the RedHatOfficial organization will get in touch with you.
+
+## Quick start
+    ```
+    git clone https://github.com/RedHatOfficial/RedHatOfficial.github.io.git
+    cd RedHatOfficial.github.io
+    
+    ```
+ 
+## Building and Running Locally
+Building and running the Red Hat Official Project Page in your local dev environment is very easy. Be sure you have [Node.js](https://nodejs.org/) installed, then follow the directions below. 
```

Comment:
```suggestion
To build and run this project in your local dev environment, follow the instructions below. Be sure you have [Node.js](https://nodejs.org/) installed before you start.
```

Try to avoid value statements like `very easy` because everyone has varying levels of experience and what is easy for one person is challenging for another.